app/models/katello/glue/pulp/repo.rb

```diff
@@ -511,6 +511,7 @@ def index_db_docker_manifests
 
       def docker_manifests_json
         docker_manifests = []
+        DockerTag.where(:repository_id => id).destroy_all
```

Comment:
Is this on purpose?  It seems out of place to destroy records in method to intended to return json.